Frontend/src/components/SuggestedUsers.jsx

```
@@ -91,6 +150,49 @@ const SuggestedUsers = () => {
+              </h3>
+              <p className="text-sm text-gray-500 text-center">
+                {req.userDetails?.bio || "No bio available"}
+              </p>
+
+              {/* Accept & Reject Buttons */}
+              <div className="flex justify-between mt-3">
+                <button
+                  onClick={() => acceptConnectionRequest(req.id)}
+                  className="w-1/2 bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded-lg mx-1"
+                >
+                  Accept
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)